assets/device/classes/lock.json

```
@@ -18,7 +20,9 @@
     "sv": "Använd den här enhetsklassen för att låsa enheter, oftast tillsammans med funktionerna `låst` och `låsläge`.",
     "no": "Bruk denne enhetsklassen for låseenheter, ofte sammen med funksjonene `låst` og `låst_modus`.",
     "es": "Utiliza esta clase de dispositivo para las cerraduras, generalmente junto con las capacidades de 'cerrado' y 'modo_cerradura'.",
-    "da": "Brug denne enhedsklasse til låseenheder, normalt sammen med `låst` and `låst_tilstand` funktionerne."
+    "da": "Brug denne enhedsklasse til låseenheder, normalt sammen med `låst` and `låst_tilstand` funktionerne.",
```

```suggestion
    "da": "Brug denne enhedsklasse til låseenheder, normalt sammen med `locked` and `lock_mode` funktionerne.",
```
This should also be fixed for the other translations in this description